Documentation for latest logout changes (#1624)
keycloak/keycloak-documentation: server_admin/topics/sso-protocols/con-oidc-auth-flows.adoc

Once the user is redirected to the logout endpoint, {project_name} is going to send logout requests to
clients to let them to invalidate their local user sessions, and potentially redirect the user to some URL

> once the logout process is finished. The user might be optionally requested to confirm the logout in case that `id_token_hint` parameter was not used.

Reviewer: ```suggestion
once the logout process is finished. The user might be optionally requested to confirm the logout in case the  `id_token_hint` parameter was not used.
```

> After logout, user will be automatically redirected to the specified `post_logout_redirect_uri` as long as it is provided as a parameter.

Reviewer: ```suggestion
After logout, the user is automatically redirected to the specified `post_logout_redirect_uri` as long as it is provided as a parameter.
```

> Note that it is needed to include either `client_id` or `id_token_hint` parameter in case that `post_logout_redirect_uri` is included. Also the `post_logout_redirect_uri` parameter

Reviewer: ```suggestion
Note that you need to include either the `client_id` or `id_token_hint` parameter in case the `post_logout_redirect_uri` is included. Also the `post_logout_redirect_uri` parameter
```

> needs to match with one of the `Valid Post Logout Redirect URIs` specified in the client configuration.

Reviewer: ```suggestion
needs to match one of the `Valid Post Logout Redirect URIs` specified in the client configuration.
```
